src/ui/widgets/ActionButton/actionButton.tsx

```diff
@@ -35,38 +39,39 @@ export interface ActionButtonProps {
 export const ActionButtonComponent = (
   props: ActionButtonProps
 ): JSX.Element => {
-  const style = commonCss(props);
-  if (props.disabled) {
-    style["cursor"] = "not-allowed";
-  }
-  style["whiteSpace"] = "normal";
-  // Use default button style if no border defined.
-  if (props.border?.style === BorderStyle.None) {
-    style["borderStyle"] = undefined;
-    style["borderWidth"] = undefined;
-    style["borderColor"] = undefined;
-    style["padding"] = "0";
-  }
   return (
-    <button
-      className={classes.actionbutton}
-      disabled={props.disabled}
-      onClick={props.onClick}
-      style={style}
-    >
-      {props.image !== undefined ? (
-        <figure className={classes.figure}>
-          <img
-            style={{ width: "100%", display: "block" }}
-            src={props.image}
-            alt={props.image}
-          ></img>
-          <figcaption>{props.text}</figcaption>
-        </figure>
-      ) : (
-        <span style={{ whiteSpace: "normal" }}>{props.text || ""}</span>
-      )}
-    </button>
+    <ThemeProvider theme={defaultColours}>
+      <Button
+        variant="contained"
+        disabled={props.disabled}
```

Comment:
This doesn't seem to be setting properly, it looks like the prop used in the .bob/.opi file is `enabled` instead of `readonly` property that is then mapped to `disabled` later on line 116. (It appears this is an existing bug in this code, but it would be good to fix it here)